analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnchangedLocalVariablesShouldBeConst.ToFix.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+class ToReplace
+{
+    void Alias()
+    {
+        string str = "str";   // Noncompliant
+        bool b = false;       // Noncompliant
+        char ch = 'c';        // Noncompliant
+        byte bits = 8;        // Noncompliant
+        sbyte sbits = 8;      // Noncompliant
+        short int16 = 16;     // Noncompliant
+        ushort uint16 = 16;   // Noncompliant
+        int int32 = 32;       // Noncompliant
+        uint uint32 = 32;     // Noncompliant
+        long int64 = 64;      // Noncompliant
+        ulong ulongVal = 1;   // Noncompliant
+        float single = 32;    // Noncompliant
+        double floating = 64; // Noncompliant
+        decimal dec = 128;    // Noncompliant
+        int i1 = 1,           // Noncompliant
+            i2 = 2;           // Noncompliant
```

Comment:
Please add a test case where we raise only at some declarators but not all and let's see what happens then.
```cs
int i3 = DateTime.Now.Day,  // Compliant
    i4 = 4,                 // Noncompliant
    i5;                     // Compliant
```

Reply:
If this breaks, you should not offer a fix if multiple declarators are present (It is an uncommon pattern and also hard to get the fix right. It is not worth all the hassle).

Reply:
Nice scenario. It is broken when you try to fix that automatically.

Note that the tooling replaces *Noncompliant* with *Fixed*. I'm not sure if that is convenient.